### DISCUSSION TITLE
Can't deploy on github pages

### DISCUSSION THREAD

**Discussion by mysubcult** [thuliteio/doks#986] [Q&A]
Hello, I used the instructions from here: https://github.com/h-enk/doks-gh-pages
But I can't do step 3 because I don't have this: "Go to the Settings tab of your GitHub repo, and next to the Pages section. Select branch gh-pages and click Save." 
P.S. https://mysubcult.github.io/doksdoks/

**Comment by h-enk** [comment]
Does this help?

![Snag_256bbe4e](https://user-images.githubusercontent.com/3902872/213186703-ab1776c2-1fae-4525-8fce-7218608f4d42.png)
